web/src/main/kotlin/no/nav/su/se/bakover/web/Application.kt

```diff
@@ -154,6 +156,20 @@ internal fun Application.susebakover(
                 )
             )
         }
+        exception<Tilgangssjekkfeil> {
```

Comment:
Er dette en generell feil, eller er dette egentlig er en spesiell type PDL-feil?

Reply:
Det er tenkt som en generell "Du har ikke tilgang pga. noe som ikke har med AD-roller å gjøre". Per nå er det i praksis kun når vi får feil fra PDL, som blir gjort her, men har laget den typen i stedet for å bruke `PdlFeil` direkte for å legge opp til enklere utviding senere.